BlockEntity/BEFruitpress.cs

```diff
@@ -601,11 +675,6 @@ public override void OnReceivedServerPacket(int packetid, byte[] data)
 
                 if (packet.AnimationActive)
                 {
-                    if (!MashSlot.Empty && juiceableLitresLeft > 0 && !CompressAnimActive)

```

Comment:
Since the server tells all clients to play the sound at a different point in time _without_ this bit it felt unnecessary and it also meant the sound played at a different time for people who weren’t the ones using the machine.